etc/DependencyInstaller.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Make sure we are on the correct folder before beginning
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    cd "$(dirname $(perl -e 'use Cwd "abs_path";print abs_path(shift)' $0))/../"
+else
+    cd "$(dirname $(readlink -f $0))/../"
+fi
+
+_installORDependencies() {
+    ./tools/OpenROAD/etc/DependencyInstaller.sh
```

Comment:
Add support for the new flags, `-base`, `-common`, etc. The ORFS installer should receive the same subset of arguments and forward them to OR installer.